exporter/datasetexporter/README.md

```diff
@@ -43,6 +43,17 @@ The process of populating the serverHost attribute works as follows:
 
 Make sure to provide the appropriate server host value in the `serverHost` attribute to ensure the proper functionality of DataSet and accurate handling of events.
 
+### Attributes
+
+If exporting of attributes of given type are enabled, then they are exported in following order:
```

Comment:
Enabled attributes are exported in the order:

Reply:
Fixed